src/templates/common/contentFooter/index.tsx

```diff
@@ -33,91 +38,59 @@ function formatDate(date: Date, format: 'display' | 'machine'): string {
   }
 }
 
-function FeedbackButton() {
-  useEffect(() => {
```

Comment:
This was just used to add the `feedback-button` class to the button, but we don't need that custom styling because we are using a web component for the button now.